### DISCUSSION TITLE
Improvement on one hand / compact mode

### DISCUSSION THREAD

**Discussion by Fenghuang19** [AnySoftKeyboard/AnySoftKeyboard#3451] [Ideas]
Can you add a gesture shortcut for disable the compact mode (I only see one for enable it) or directly an additional togglable button on the keyboard for enter, exit this mode.

Also it will be great if there will be an option for resize the dimensions of the keyboard on compact mode.

Thank's in advance

**Comment by nicoursi** [comment]
Shortcuts already exist, you just need to activate them from the settings.

![Screenshot_20230314-212327_AnySoftKeyboard](https://user-images.githubusercontent.com/16888339/225127918-1c2b9815-4165-4672-8f64-50ecc78d5bb7.png)


https://user-images.githubusercontent.com/16888339/225128236-8ea43681-85cc-4ea1-a0aa-389f6ff2aa94.mp4

**Reply by Fenghuang19** [reply]
Ok, I get it for the shortcut so the only missing feature is to be able to resize the dimensions of the keyboard on compact mode.
